### ISSUE TITLE
Importer throws NullPointerException on Model Import

### ISSUE THREAD

**Issue by daniel-de** [3dcitydb/importer-exporter#103]
Hi,

trying to import a file with Version 4.2.2, which is imported successfully with Version 3.3.2 now throws a NullPointerException after xAL address element:
```
[16:05:38 ERROR] core:Address (unknown gml:id): Failed to interpret xAL address element.
[16:05:38 ERROR] An error occurred: null
[16:05:38 ERROR] Aborting due to an unexpected java.lang.NullPointerException error.
```
I could share a small data set for debugging if necessary...

kind regards,
Daniel

**Comment by clausnagel** [comment]
Hi Daniel, a test dataset would indeed be very helpful.

**Comment by daniel-de** [comment]
Hi Claus,

I'm not allowed to post the data but here.
Sending them via email?

**Comment by clausnagel** [comment]
Sure. You can use the email from my GitHub profile: cnagel@virtualcitysystems.de

**Comment by clausnagel** [comment]
Your dataset violates the CityGML schema. For two addresses, the `<xAL:AddressDetails>` element is given as direct child element of `<core:Address>`. However, there must be a `<core:xalAddress>` property element inbetween (check lines 220340 and 220364). You can simply validate your CityGML input datasets by clicking the "Just validate" button on the "Import" tab. Interestingly, all other address elements in your dataset are correct.

Nevertheless, there was also a bug which caused the import process to fail for invalid address elements. I fixed this bug (3825ebb2991d0329028dcd0f7acfaba73bb88076). Simply build a new version of the importer from the `master` branch to benefit from the bugfix.

**Comment by daniel-de** [comment]
Thanks a lot, dataset is importing now. The model was exported from proprietary software. Trying to inform the manufacturer about the bug...
